docs/security/certificate-management.md

```diff
@@ -690,9 +191,9 @@ Setting CA Certificates for at '$HOME/certificates'
 Setup completed successfully.
 ```
 
-#### Adding/Removing a directory of local CAs ####
+#### Adding/Removing a directory of local CAs
 
-##### Adding #####
+##### Adding
```

Comment:
Let's turn this section and its parent section into an h4 named `Adding a directory of local CAs`